arduino/cores/packagemanager/install_uninstall.go

```diff
@@ -313,7 +313,8 @@ func (pme *Explorer) InstallTool(toolRelease *cores.ToolRelease, taskCB rpc.Task
 
 	toolResource := toolRelease.GetCompatibleFlavour()
 	if toolResource == nil {
-		return fmt.Errorf(tr("no compatible version of %s tools found for the current os"), toolRelease.Tool.Name)
+		return fmt.Errorf(tr("no compatible version of %s tools found for the current os, try contacting %s"),
+			toolRelease.Tool.Name, toolRelease.Tool.Package.Email)
```

Comment:
This string can not be correctly translated, you must replace unindexed `%s` with `%[1]s` and `%[2]s`.

Reply:
Modified here: https://github.com/arduino/arduino-cli/pull/2099